OxylusEditor/Resources/Shaders/passes/bloom/bloom_downsample.slang

```diff
@@ -0,0 +1,31 @@
+module bloom_downsample;
+
+import common;
+import gpu;
+
+import passes.bloom.bloom;
+
+[[vk::binding(0, 0)]]
+StorageImage2D<f32x4, ImageFormat::RGBA16F> result_image;
+[[vk::binding(1, 0)]]
+Image2D<f32x4> source_image;
+[[vk::binding(2, 0)]]
+SamplerState sampler;
```

Comment:
We should use `ParameterBlock<T>` here.
In single stage shaders, parameter blocks should be passed inside entry point function args (this is valid for current case because it's compute) like this:

```slang
[shader("compute")]
func cs_main(uniform ParameterBlock<ShaderParameters> params) -> void { ... }
```

for multi-stage (vertex-fragment) shaders it's okay to define `ParameterBlock<T>` inside global scope.